src/clusterfuzz/_internal/google_cloud_utils/storage.py

```diff
@@ -89,6 +90,12 @@
 OBJECT_URL = 'https://storage.cloud.google.com'
 DIRECTORY_URL = 'https://console.cloud.google.com/storage/browser'
 
+# Expiration in minutes for signed URL.
+SIGNED_URL_EXPIRATION_MINUTES = 24 * 60
+
+# Timeout for HTTP operations.
+HTTP_TIMEOUT_SECONDS = 15
```

Comment:
I am unfamiliar with the context here, but I guess we do not expect this value to change over time or vary between projects?